### PR TITLE
Add button to open gui in browser

### PR DESCRIPTION
**Pull request by WeirdRubberDuck**
And add some newlines to make the divisions in the menu a bit cleare

closes OpenSpace/OpenSpace#2337

https://github.com/OpenSpace/OpenSpace-WebGuiFrontend/assets/8808894/dfaab1cf-db9a-4ae7-b753-007e219eca6e

### CODE REVIEW

File: src/components/SystemMenu/SystemMenu.jsx [hunk 1]
```diff
@@ -65,6 +65,18 @@ function SystemMenu({ showTutorial }) {
     api.executeLuaScript(script);
   }
 
+  async function openGuiInBrowser() {
+    if (!luaApi) { return; }
+    const portProperty = await luaApi.propertyValue('Modules.WebGui.Port');
```
Comment (ylvaselling) [inline src/components/SystemMenu/SystemMenu.jsx:70]: If this is a property, shouldn't we already have it in the property tree? Might be more performant to use useSelector than to wait for the engine to pass the value

File: src/components/SystemMenu/SystemMenu.jsx [hunk 2]
```diff
@@ -105,13 +117,16 @@ function SystemMenu({ showTutorial }) {
             <button type="button" onClick={() => onClick(openFeedback)}>
               Send Feedback
             </button>
+
```
Comment (ylvaselling) [inline src/components/SystemMenu/SystemMenu.jsx:120]: Why new lines?

File: src/components/SystemMenu/SystemMenu.jsx [hunk 3]
```diff
@@ -122,20 +137,30 @@ function SystemMenu({ showTutorial }) {
...
             <button type="button" onClick={() => { onClick(nativeGui); }}>
               Toggle native GUI
               {' '}
               <span className={styles.shortcut}>F1</span>
             </button>
-            {/*              <button onClick={saveChange}>
+
+            {/* <button onClick={saveChange}>
```
Comment (ylvaselling) [inline src/components/SystemMenu/SystemMenu.jsx:158]: Not sure if we want to keep this comment?

**Review by ylvaselling — APPROVED** [review]

File: src/components/SystemMenu/SystemMenu.jsx (hunk 2 above)
Comment (WeirdRubberDuck) [reply src/components/SystemMenu/SystemMenu.jsx:120]: Found it a bit hard to see where the separated sections started and ended as the `<HorizontalDelimiter />` is on the same indention level as all the buttons. This was a whim to make that a bit easier

But I see below that I wasnät even consistent. I'm ok with removing them again

File: src/components/SystemMenu/SystemMenu.jsx (hunk 3 above)
Comment (WeirdRubberDuck) [reply src/components/SystemMenu/SystemMenu.jsx:158]: I really don't know either.  It shows the intention of bringing back the button for saving changes to the profile, but at this stage I don't even know if we want to. 

I would be ok with removing it

File: src/components/SystemMenu/SystemMenu.jsx (hunk 1 above)
Comment (WeirdRubberDuck) [reply src/components/SystemMenu/SystemMenu.jsx:70]: Sure.  Just did it the same way here as we do a bit further up in the code, for consistency. But I'm ok with changing it to a using useSelector for the other places in this file as well

**Pull request merged**